shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Tick.hs

```diff
@@ -126,3 +142,45 @@ instance
   Embed (RUPD era) (TICK era)
   where
   wrapFailed = RupdFailure
+
+{------------------------------------------------------------------------------
+-- TICKF transition
+
+-- This is a variant on the TICK transition called only by the consensus layer
+to tick the ledger state to a future slot.
```

Comment:
ledger state -> `ChainDepState`

Reply:
No, there's a separate thing for the chain dep state. This is used to tick the ledger state for projection to the ledger view.